mlrun/utils/notifications/notification_pusher.py

```diff
@@ -226,15 +226,19 @@ def _should_notify(
         for when_state in when_states:
             if when_state == run_state:
                 if (
-                    run_state == "completed"
+                    run_state == runtimes_constants.RunStates.completed
                     and evaluate_condition_in_separate_process(
                         notification.condition,
                         context={
                             "run": run.to_dict(),
                             "notification": notification.to_dict(),
                         },
                     )
-                ) or run_state in ["error", "aborted"]:
+                ) or run_state in [
+                    runtimes_constants.RunStates.error,
+                    runtimes_constants.RunStates.aborted,
+                    runtimes_constants.RunStates.running,
```

Comment:
why adding running here?

Reply:
We want to  make running notification also send from the server side.